src/main/kotlin/app/CodeLongevity.kt

```diff
@@ -0,0 +1,173 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+
+package app
+
+import org.eclipse.jgit.blame.BlameGenerator
+import org.eclipse.jgit.blame.BlameResult
+import org.eclipse.jgit.diff.DiffFormatter
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.diff.RawText
+import org.eclipse.jgit.internal.storage.file.FileRepository
+import org.eclipse.jgit.lib.Repository
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import org.eclipse.jgit.revwalk.RevSort
+import org.eclipse.jgit.treewalk.filter.*
+import org.eclipse.jgit.treewalk.TreeWalk
+import org.eclipse.jgit.util.io.DisabledOutputStream
+
+import java.util.Date
+
+/**
+ * Used to store and identify a code line information in a repo.
+ */
+class CodeLine(val rev: String, val file: String, val line: Int,
+               val authorName: String, val authorEmail: String,
+               val age: Int) {
+}
+
+/**
+ * Used to compute age of code lines in the repo.
+ */
+class CodeLongevity(repo_path: String) {
+    var repo_path: String = repo_path
+
+    var list: MutableList<CodeLine> = mutableListOf()
+
+    init {
+        println("READYY>>>>>>>>>>");
```

Comment:
Well, I was asking for a feedback, not for a review for sure. Feel sorry for not making it clear from the start, it would save your time.